spyder/plugins/pylint/widgets/pylintgui.py

```diff
@@ -310,8 +312,16 @@ def show_log(self):
             TextEditor(self.output, title=_("Pylint output"),
                        readonly=True, size=(700, 500)).exec_()
 
+    @Slot()
+    def analyze_button_handler(self):
+        """ Try to start code analysis when Analyze button pressed."""
+
+        self.start_analysis.emit()
+
     @Slot()
     def start(self):
+        """ Start the code analysis. """

```

Comment:
Please remove the blank spaces around the docstring, like this:

    """Start the code analysis."""